src/services/upper-bar-manager/ui/dropdown-bar/dropdown-bar.component.tsx

```diff
@@ -10,10 +10,10 @@ type DropdownBarProps = {
 };
 
 export class DropdownBar extends Component<DropdownBarProps> {
-  private handleOnKeyDown(event: KeyboardEvent, onClick: () => void): void {
+  private handleOnKeyDown(event: KeyboardEvent, onClick: (e: MouseEvent | KeyboardEvent) => void): void {
```

Comment:
onClick with keyboard ?

Reply:
this is not native onClick, this is IconModel onClick which is Handel both keyboards and mouseclick
like you can see in line 15